cfme/automate/buttons.py

```diff
@@ -853,6 +855,16 @@ def delete_all(self):
         for btn_grp in self.appliance.rest_api.collections.custom_button_sets.all:
             btn_grp.action.delete()
 
+    def instantiate_with_rest(self, id):
+        from cfme.tests.automate.custom_button import CLASS_MAP
+        gp, *_ = self.appliance.rest_api.collections.custom_button_sets.find_by(id=id)
+        rest_btn_type = gp.set_data.get("applies_to_class")
+        for maps in CLASS_MAP.values():
+            if maps.get("rest") == rest_btn_type:
+                ui_btn_type = maps.get("ui")
+                break
```

Comment:
**Question:**
@digitronik can you please tell me why explicit use of `break`, any how it will return from `if` block .?